server/routes/termdb.topVariablyExpressedGenes.ts

```diff
@@ -35,10 +35,24 @@ function init({ genomes }) {
 			if (!ds.queries?.topVariablyExpressedGenes) throw 'not supported on dataset'
 
 			const t = Date.now()
-			const genes = await ds.queries.topVariablyExpressedGenes.getGenes(q)
+			let genes = await ds.queries.topVariablyExpressedGenes.getGenes(q)
+
+			const foundGenes = new Set()
```

Comment:
q.geneSet should be a param to `getGenes()` rather than subsequent filter. i will fix this